app/views/proposal/proposal_partials/_manage_proposals_approver.html.erb

```diff
@@ -0,0 +1,55 @@
+<section class="eui-callout-box col-left">
+  <div class="open-drafts upcoming-proposals">
+    <h3 class="eui-callout-box__title slate-grey">Upcoming Proposals</h2>
+    <div class="eui-callout-box__list">
+      <ul>
+        <% unless @in_work_proposals.any? %>
+          There are no draft proposals to display.
+        <% end %>
+        <% @in_work_proposals[0..@proposal_display_max_count-1].each do |proposal| %>
+          <li>
+            <p class="drafts">
+              <%= time_tag proposal.updated_at, proposal.updated_at.to_s(:date) %> |
+              <%= link_to proposal.display_short_name, send("#{@proposal_type.underscore}_path", proposal), title: proposal.display_short_name %>
+              <br>
+              <%= status_content_tag(proposal) %>
+            </p>
+          </li>
+        <% end %>
+        <% if @in_work_proposals.size > @proposal_display_max_count %>
+          <li>
+            <%= link_to 'More', send("upcoming_index_#{@proposal_type.underscore}s_path") %>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="eui-callout-box col-left">
+  <div class="open-drafts queued-proposals">
+    <h3 class="eui-callout-box__title slate-grey">Queued Proposals</h2>
```

Comment:
as I mentioned, I lean towards the Queue being on the left, but that is ultimately a PO call

Reply:
PO said middle